build/resources/main/templates/login.html

```diff
@@ -4,48 +4,30 @@
 <body>
 <header th:replace="~{fragments :: header}"></header>
 <main>
-    <div class="login text-center">
+    <div class="login">
```

Comment:
Looks like this login is from a previous merge if you want to update it to the newest one 
e.g. starting on line 7 it says:      <div class="login text-center">